app/javascript/icons/CogIcon.jsx

```diff
@@ -0,0 +1,15 @@
+import { h } from 'preact';
+
+export const CogIcon = () => (
+  <svg
```

Comment:
off topic question, can these be files on disk eventually?

Reply:
PLEASE!